types/markdown-it-source-map/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for markdown-it-source-map 0.1
+// Project: https://github.com/tylingsoft/markdown-it-source-map#readme
+// Definitions by: Felix Wotschofsky <https://github.com/feliskio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import MarkdownIt = require('markdown-it');
+
+declare const markdownItSourceMap: MarkdownIt.PluginSimple;
+export = markdownItSourceMap;
```

Comment:
It looks like the original package exports this via a `default` export. Could you update this? Thanks.

Reply:
You would need to update the test to use a default import, instead of `require`, like it is on the README you linked to:

```ts
import markdownIt from 'markdown-it'
import markdownItSourceMap from 'markdown-it-source-map'
 
const mdi = markdownIt()
mdi.use(markdownItSourceMap)
 
mdi.render('# hello world') // <h1 data-source-line="1">hello world</h1>
```

If you do that and change the export to be a default export, then I think the test will also pass.